.github/workflows/tag.yml

```diff
@@ -0,0 +1,26 @@
+name: Release
+# This workflow helps with creating releases.
+# This job will only be triggered when a tag (vX.X.x) is pushed
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[12].[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
I don't think this matches `v20.15.10`, `v[12]` matches one character (either 1 or 2).
Why not matching `"v[0-9]+.[0-9]+.[0-9]+"` ?

Reply:
yup youre right it would have only run on major versions 1 & 2